package.json

```diff
@@ -32,6 +32,7 @@
     "copy-to-clipboard": "^3.2.0",
     "date-fns": "^2.0.0-beta.4",
     "dotenv": "^8.0.0",
+    "file-loader": "^6.2.0",
```

Comment:
It sounds a bit like `file-loader`, `ts-loader`, and `utility-types` should be devDependencies